Rakefile

```
@@ -1,9 +1,20 @@
 require 'rspec/core/rake_task'
 
-namespace :test do
+namespace :lint do
+  require 'rubocop/rake_task'
+  RuboCop::RakeTask.new(:rubocop) do |task|
+    # Use Rubocop's Github Actions formatter if possible
+    if ENV['GITHUB_ACTIONS'] == 'true'
+      rubocop_spec = Gem::Specification.find_by_name('rubocop')
+      if Gem::Version.new(rubocop_spec.version) >= Gem::Version.new('1.2')
+        task.formatters << 'github'
+      end
```

You pin to a newer version than 1.2 so it's safe to simplify this
```suggestion
      task.formatters << 'github'
```